docs/config-file-glossary.rst

```diff
@@ -97,14 +97,7 @@ Imgur settings
 ``USE_IMGUR_FOR_IMAGES=true``
     Upload picture messages from Telegram to Imgur, send Imgur link to IRC
 
-``IMGUR_CLIENT_ID=0000000000``
-    Imgur API client ID value to access Imgur API
-
-
-**********************
-Miscellaneous settings
-**********************
-
-``NTBA_FIX_319=1``
-    Required to fix a bug in a library used by TeleIRC.
-    For context, see `yagop/node-telegram-bot-api#319 <https://github.com/yagop/node-telegram-bot-api/issues/319#issuecomment-324963294>`_.
+``IMGUR_CLIENT_ID=7d6b00b87043f58``
+    Imgur API client ID value to access Imgur API.
+    Uses a default API key.
```

Comment:
Not a blocker, but maybe mentioning this is TeleIRC's general API key?

Reply:
@Tjzabel It's mentioned here: https://github.com/RITlug/teleirc/pull/279/files#diff-82fe77d984d3c73e3c8cd6c3216a3ce2R67

Do you think this is sufficient?